app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights to return per page'
+    end
+    property :order do
+      key :type, :string
+      key :enum, %w[asc desc]
+      key :description, 'The returned sorted order'
```

Comment:
Say the order is based on whatever fields those are in the Highlight.

Specify the default if there is one.